narwhals/_pandas_like/expr.py

```diff
@@ -294,6 +294,9 @@ def head(self, n: int) -> Self:
     def tail(self, n: int) -> Self:
         return reuse_series_implementation(self, "tail", n)
 
+    def round(self: Self, decimals: int = 0) -> Self:
```

Comment:
i should add a pre-commit rule here, but i think we should avoid default args in _pandas_like
```suggestion
    def round(self: Self, decimals: int) -> Self:
```